perfkitbenchmarker/providers/azure/azure_kubernetes_service.py

```
@@ -392,6 +392,89 @@ def GetNodePoolNames(self) -> list[str]:
     nodepools = json.loads(stdout)
     return [nodepool['name'] for nodepool in nodepools]
 
+class AksAutomaticCluster(AksCluster):
+  """Class representing an AKS Automatic cluster, which has managed node pools."""
+
+  CLOUD = provider_info.AZURE
+  CLUSTER_TYPE = 'Auto'
+
+  def __init__(self, spec):
+    """Initializes the cluster."""
+    super().__init__(spec)
+
+  def _Create(self):
+    """Creates the Automatic AKS cluster with tags."""
+    tags_dict = util.GetResourceTags(self.resource_group.timeout_minutes)
+    tags_list = [f'{k}={v}' for k, v in tags_dict.items()]
+    cmd = [
+        azure.AZURE_PATH,
+        'aks',
+        'create',
+        '--name',
+        self.name,
+        '--location',
+        self.region,
+        '--ssh-key-value',
+        vm_util.GetPublicKeyPath(),
+        '--resource-group',
+        self.resource_group.name,
+        '--sku',
+        'automatic',
+        '--tags',
+    ] + tags_list
+    vm_util.Retry(timeout=300)(vm_util.IssueCommand)(
```

Why retry here? what errors are being encountered & ignored but are fixed on a subsequent aks create? Failure rather than swallowing errors is sometimes preferred.

..ah I see this code matches AksCluster.. which also means I'm guessing you haven't run into the race condition mentioned above yet? I'd remove it atm & we can add it back if we run into the aforementioned race condition.